test/e2e/user_input_test.go

```diff
@@ -141,3 +107,43 @@ func checkUsersAttributes(input userInputs) {
 		}
 	}
 }
+
+func PrepareResoures(input model.UserInputs) {
+	CopyKustomizeNamespaceOperator(input)
+	CopyTestAppForEachUser(input)
+}
+
+// CopyKustomizeNamespaceOperator create copy of `/deploy/namespaced` folder with kustomization file for overriding namespace
+func CopyKustomizeNamespaceOperator(input model.UserInputs) {
+	// fullPath := filepath.Join("data", input.projectName, "operator")
+	fullPath := input.GetOperatorFolder()
+	// fullPath := filepath.Dir(projectPath)
+	os.Mkdir(fullPath, os.ModePerm)
+	utils.CopyFile("../../deploy/namespaced/crds.yaml", filepath.Join(fullPath, "crds.yaml"))
+	utils.CopyFile("../../deploy/namespaced/namespaced-config.yaml", filepath.Join(fullPath, "namespaced-config.yaml"))
+	data := []byte(
+		"namespace: " + input.Namespace + "\n" +
+			"resources:" + "\n" +
+			"- crds.yaml" + "\n" +
+			"- namespaced-config.yaml",
+	)
+	utils.SaveToFile(filepath.Join(fullPath, "kustomization.yaml"), data)
+}
+
+// each user has his own application?
+func CopyTestAppForEachUser(input model.UserInputs) {
+	fullPath := input.GetAppFolder()
+	for _, user := range input.Users {
```

Comment:
not sure I understand why we copy the configs to the other directories - I thought we use helm instead?

Reply:
I will look, probably wasn't clean it after init verson